spec/services/proofing/resolution/progessive_proofer_spec.rb

```diff
@@ -0,0 +1,236 @@
+        end
+
+        context 'user is in an AAMVA jurisdiction' do
+          let(:resolution_result_that_passed_instant_verify) do
+            instance_double(Proofing::Resolution::Result)
+          end
+
+          before do
+            allow(instance).to receive(:proof_resolution).
+              and_return(resolution_result_that_passed_instant_verify)
+            allow(instant_verify_proofer).to receive(:proof).with(hash_including(state_id_address)).
+              and_return(resolution_result_that_passed_instant_verify)
+            allow(instance).to receive(:user_can_pass_after_state_id_check?).
+              with(resolution_result_that_passed_instant_verify).
+              and_return(true)
+            allow(resolution_result_that_passed_instant_verify).to receive(:success?).
+              and_return(true)
+          end
+
+          it 'makes a request to the AAMVA proofer' do
+            expect(aamva_proofer).to receive(:proof)
+
+            subject
+          end
+
+          context 'AAMVA proofing fails' do
+            let(:aamva_client) { instance_double(Proofing::Aamva::VerificationClient) }
+            let(:failed_aamva_proof) do
+              instance_double(Proofing::StateIdResult)
+            end
+            before do
+              allow(Proofing::Aamva::VerificationClient).to receive(:new).and_return(aamva_client)
+              allow(failed_aamva_proof).to receive(:success?).and_return(false)
+            end
+            it 'returns a result adjudicator that indicates the aamva proofing failed' do
+              allow(aamva_proofer).to receive(:proof).and_return(failed_aamva_proof)
+
+              result = subject
+
+              expect(result.state_id_result.success?).to eq(false)
+            end
+          end
+        end
+      end
+
+      context 'Instant Verify fails' do
+        let(:aamva_proofer) { instance_double(Proofing::Aamva::Proofer) }
+        let(:result_that_failed_instant_verify) do
+          instance_double(Proofing::Resolution::Result)
+        end
+
+        before do
+          allow(instance).to receive(:state_id_proofer).and_return(aamva_proofer)
+          allow(instance).to receive(:proof_resolution).
+            and_return(result_that_failed_instant_verify)
+          allow(instant_verify_proofer).to receive(:proof).with(hash_including(state_id_address)).
+            and_return(result_that_failed_instant_verify)
+        end
+
+        context 'the failure can be covered by AAMVA' do
+          before do
+            allow(instance).to receive(:user_can_pass_after_state_id_check?).
+              with(result_that_failed_instant_verify).
+              and_return(true)
+            # rubocop:disable Layout/LineLength
+            allow(result_that_failed_instant_verify).to receive(:attributes_requiring_additional_verification).
+              and_return([:address])
+            # rubocop:enable Layout/LineLength
```

Comment:
I would add a line break before the `to`
```suggestion
            allow(result_that_failed_instant_verify).
              to receive(:attributes_requiring_additional_verification).and_return([:address])
```

Reply:
Good idea. 👍🏻